modules/Layout/Header/SearchWidget/components/SearchBar.tsx

```diff
@@ -37,30 +36,7 @@ function SearchBar({ currentRefinement, refine }: Props) {
                         <FormattedMessage
                             {...translations.search.inputHint}
                             values={{
-                                inputHintExtra: (
-                                    <span className={styles.inputHintDesktop}>
-                                        <FormattedMessage
-                                            {...translations.search.inputHintExtra}
-                                            values={{
-                                                keyHint: (
-                                                    <span className={styles.keyHint}>
-                                                        <IconEnterKey className={styles.keyIcon} />
-                                                        <span>Enter</span>
-                                                    </span>
-                                                ),
-                                                buttonLabel: (
-                                                    <>
-                                                        &quot;
-                                                        <FormattedMessage
-                                                            {...translations.search.action}
-                                                        />
-                                                        &quot;
-                                                    </>
-                                                ),
-                                            }}
-                                        />
-                                    </span>
-                                ),
+                                inputHintExtra: <span>&nbsp;</span>,
```

Comment:
Apparently, we still need to specify this. 

We could actually do something like this: 
```js
inputHintExtra: <>&nbsp;</>
```

Reply:
It can be `undefined` as well.
But ideally we would want to remove the string from `themes-intl-messages`, so that this parameter is not needed anymore.

Reply:
So a proper thing to do would be either to delete the {inputHintExtra} from `themes-intl-messages` or to delete the 
`search.inputHintExtra` in the crowdin editor?